### ISSUE TITLE
Analytic units go to LEARNING state by themselves

### ISSUE THREAD

**Issue by amper43** [hastic/hastic-server#826]
Sometimes analytic units with enabled webhooks go to LEARNING state by themselves. Probably it happened after down/up of analytics.

**Comment by jonyrock** [comment]
Yeah, I saw that problem. @rozetko do you see anythin like that in our metrics?

**Comment by rozetko** [comment]
haven't seen that since webhooks integration
